pkg/distributors/contracts/MultiRewards.sol

```diff
@@ -72,36 +73,30 @@ contract MultiRewards is IMultiRewards, IDistributor, ReentrancyGuard, Temporari
 
     /* ========== CONSTRUCTOR ========== */
 
-    constructor(IVault _vault) Ownable() TemporarilyPausable(3600, 3600) {
-        vault = _vault;
+    constructor(IVault _vault)
+        /* disambiguate based on who is deploying staking contract */
+        Authentication(bytes32(uint256(msg.sender)))
+        MultiRewardsAuthorization(_DELEGATE_OWNER, _vault)
+        TemporarilyPausable(3600, 3600)
```

Comment:
1 hour seems very short, is there much benefit to this?

Reply:
good question (didn't change the pause window in this PR) what do you think it should be?  24 hours?

Reply:
I think this should be at least a couple of rounds of rewards so maybe a month or so?